src/config.rs

```diff
@@ -148,7 +149,8 @@ impl Config {
             prefix: prefix.clone(),
 
             database_url: require_env("DOCSRS_DATABASE_URL")?,
-            max_pool_size: env("DOCSRS_MAX_POOL_SIZE", 90)?,
+            max_legacy_pool_size: env("DOCSRS_MAX_LEGACY_POOL_SIZE", 45)?,
+            max_pool_size: env("DOCSRS_MAX_POOL_SIZE", 45)?,
```

Comment:
Will we ever want to actually configure these separately, or could we just halve it between the two pools?

Reply:
I'm not 100% certain, 

in my mind, depending on how much is migrated and how much is not, we might need different pool sizes.